src/homeform.cpp

```diff
@@ -36,15 +36,24 @@ using namespace std::chrono_literals;
 #include <QtAndroid>
 #endif
 
-#include "secret.h"
-
-#ifndef STRAVA_CLIENT_ID
-#define STRAVA_CLIENT_ID 7976
-#if defined(WIN32)
-#pragma message("DEFINE STRAVA_CLIENT_ID!!!")
+#if __has_include("secret.h")
+#   include "secret.h"
 #else
-#warning "DEFINE STRAVA_CLIENT_ID!!!"
+#   define STRAVA_SECRET_KEY test
+#   if defined(WIN32)
+#       pragma message("DEFINE STRAVA_SECRET_KEY!!!")
+#   else
+#       warning "DEFINE STRAVA_SECRET_KEY!!!"
+#   endif
 #endif
+
+#ifndef STRAVA_CLIENT_ID
+#   define STRAVA_CLIENT_ID 7976
```

Comment:
I thought it might be a good idea to move this actual value to `secret.h` to make it more obvious that Strava won't work but left it to retain your workflow. Do you think this would be a good idea to move or was there a reason you left this out of `secret.h`?

Reply:
yes you're totally right!